forte/data/data_store.py

```diff
@@ -247,6 +247,59 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         entry += len(self._type_attributes[type_name]) * [None]
         return entry
 
+    def _is_subclass(
+        self, type_name: str, cls, include_onto_file: bool = False
+    ) -> bool:
+        r"""This function determines if a string class_type is
+        the subclass of a class.
+
+        Args:
+            type_name: A fully qualified name of an entry class
+            cls: A class.
+            include_onto_file: A boolean value whether the class is in user provided ontology file
+
+        Returns:
+            A boolean value whether this `type_name` class is the subclass of `cls` or not.
+
+        """
+
+        if include_onto_file:
+            if type_name in self._type_attributes["parent_class"].keys():
+                if (
+                    cls.__module__ + "." + cls.__name__
```

Comment:
We should use utilities instead of writing this again: https://github.com/asyml/forte/blob/master/forte/utils/utils.py#L114